File: Source/C#/RiceDoctor/RiceDoctor.OntologyManager/Models/EntityType.cs
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;

namespace RiceDoctor.OntologyManager
{
    [JsonConverter(typeof(StringEnumConverter))]
    public enum EntityType
    {
        Class,
        Relation,
        Attribute,
        Individual
    }
}

File: Source/Java/src/main/java/vn/edu/uit/ontologymanager/model/EntityType.java
package vn.edu.uit.ontologymanager.model;

import com.google.gson.annotations.SerializedName;

public enum EntityType {
    @SerializedName("Class")
    CLASS,

    @SerializedName("Relation")
    RELATION,

    @SerializedName("Attribute")
    ATTRIBUTE,

    @SerializedName("Individual")
    INDIVIDUAL
}
